platform/viewer/src/components/StudyList/studyListMock.json

```diff
@@ -0,0 +1,826 @@
+{
```

Comment:
This should probably be in `@ohif/ui`. The in-memory repository stuff and the new standalone JSON format make mocking for tests in `viewer` possible.